server/tso.go

```diff
@@ -31,9 +31,12 @@ func getTimestampPath(rootPath string) string {
 
 func (s *Server) loadTimestamp() (int64, error) {
 	data, err := getValue(s.client, getTimestampPath(s.cfg.RootPath))
-	if err != nil || data == nil {
+	if err != nil {
```

Comment:
Seems more short, but in TiDB project, we have always checked the error first, then the value. So i think it is better to keep the same with that.